### PR TITLE
Fall Comp schedule

### PR DESCRIPTION
**Pull request by viviehn**

### CODE REVIEW

File: _sass/_styles.scss [hunk 1]
```diff
@@ -539,7 +539,10 @@ Does not work on round things. */
   margin: 0 auto;
 }
 
-
+.panel-schedule {
+	width: 65%;
```
Comment (scottnuma) [inline _sass/_styles.scss]: 65% becomes kinda squeezy on mobile

File: programs/rc-fall.html [hunk 2]
```diff
@@ -6,27 +6,110 @@
...
-    <h2> A Two-Part Event </h2>
-    <div class="col-md-6">
-      <div class="notecard" style="float:right">
-      <div class="big-txt"> <i class="fa fa-cogs"></i> </div>
-      <h3>RC: Reboot</h3>
+<div class="container-fluid">
+  <ul class="nav nav-tabs">
+    <li class="active"><a href="#about" data-toggle="tab" aria-expanded="false">About</a>
+    <li class=""><a href="#external" data-toggle="tab" aria-expanded="false">External Information</a></li>
+    <li class=""><a href="#internal" data-toggle="tab" aria-expanded="false">Internal</a></li>
```
Comment (jiangdaniel) [inline programs/rc-fall.html]: Last Spring, one of the things that Tommy told me to remove from the Kickoff public page was the link to the staff internal page.  His reasoning was that he believed that we should blackbox internal PiE operations from the public.  Just something to consider

File: programs/rc-fall.html [hunk 3]
```diff
@@ -6,27 +6,110 @@
...
+  </ul>
+
+  <div class="tab-content">
+    <div class="tab-pane fade active in" id="about">
+  	<div class="row text-center">
+    	<h2> A Two-Part Event </h2>
+    	<div class="col-md-6">
+      	<div class="notecard" style="float:right">
+      		<div class="big-txt"> <i class="fa fa-cogs"></i> </div>
```
Comment (scottnuma) [inline programs/rc-fall.html]: These images are nice touches --

File: programs/rc-fall.html [hunk 4]
```diff
@@ -6,27 +6,110 @@
...
+	    <th scope="row">5:00 PM</th>
+	    <td colspan="2">Close</td>
+	</tr>
+</table>
+</div>
+</div>
+</div>
+</div>
+<div class="tab-pane fade in" id="internal">
+PiE staff members, please check back here later for the internal schedule, staffing assignments, etc.
```
Comment (scottnuma) [inline programs/rc-fall.html]: This div is missing spacing on top

**Comment by scottnuma** [comment]
Lol - guess the approve changes doesn't go to look at changes. Still have more to learn about this flow

File: _sass/_styles.scss (hunk 1 above)
Comment (viviehn) [inline _sass/_styles.scss]: Fixed, it's 100% on mobile

File: programs/rc-fall.html (hunk 4 above)
Comment (viviehn) [inline programs/rc-fall.html]: Removed tab

**Comment by viviehn** [comment]
Changes to the Announcements on index.html also slipped onto this PR, my bad

**Pull request merged**